docs/versioned_docs/version-0.12.2/general/02-Getting Started/getting-started-linux.md

```diff
@@ -12,22 +12,43 @@ pagination_next: general/Getting Started/write-your-first-instruction
 - A text editor/IDE like [Visual Studio Code](https://code.visualstudio.com/)
 - :warning: [Information for Wayland](../07-Troubleshooting/linux.md#wayland).
 
-## Access Token
+## Installation
 
-As we need to prevent misuse of our API, we need you to create some credentials through our __AskUI Studio__ (for free).
+Open a directory where you would like to place the AskUI setup. If you haven’t already, set up an NPM project by running:
 
-Please [fill out this form](https://xa5a040gvfz.typeform.com/to/Ndh2NkV6) to schedule a demonstration or [request a trial](https://xa5a040gvfz.typeform.com/to/IHdr0qY5) to obtain access to __AskUI Studio__.
+```shell
+npm init -y
+```
 
-## Installation
+This will create a `package.json` file inside your present working directory which contains the configuration of your project including a description of its dependencies.
 
-Open a directory where you would like to place the AskUI setup and run this command. Answer all the questions:
+Install `askui` and other dependencies for writing and executing tests:
 
 ```shell
-npx askui@latest init -s linux
+npm i -D askui @askui/askui-reporters typescript ts-node @types/jest ts-jest jest
```

Comment:
Would probably add the missing allure environment dependency of us as otherwise the only thing that works out of the box in `@askui/askui-reporters` is the annotation reporter...